src/components/VTeleport/VTeleportTarget.vue

```diff
@@ -1,16 +1,23 @@
 <script>
 import { defineComponent } from '@nuxtjs/composition-api'
 import { targets } from './meta/targets'
+import { warn } from '@/utils/warn'
 
 export default defineComponent({
   name: 'VTeleportTarget',
   props: {
     name: { type: String, required: true },
+    element: { type: String, default: 'div' },
   },
   data: () => ({ children: [] }),
   created() {
-    if (this.name in targets)
-      throw new Error(`VTeleportTarget: duplicate name ${this.name}`)
+    console.log('created')
+    if (this.name in targets) {
+      warn(
+        `VTeleportTarget: duplicate name ${this.name}, deleting previous teleport`
+      )
+      delete targets[this.name]
```

Comment:
I wonder if this actually makes sense as a warning or if it should throw... duplicate teleport targets seems like a programming error that we wouldn't want to accidentally let into production.